### ISSUE TITLE
Mouse is not captured by the Window, making 360° rotation impossible on fullscreen and 90° difficult on default resolution

### ISSUE THREAD

**Issue by Jukitsu** [StartForKillerMC/JavaMinecraft#4]
The cursor still moves freely when moving the mouse despite having clicked on the window, making for instance 360° rotation impossible on fullscreen. This does not affect gameplay that much, it is annoying nevertheless.

**Comment by StartForKiller** [comment]
Yeah, i know, it have planned to rework the input system a bit, i'm just centering my attention to other àrts in the game, like implementing the skylight lightning
